### PR TITLE
Static route expiry if not specified as persistent 

### PR DESCRIPTION
**Pull request by prsunny**
By default, the static route is not set as persistent and saved to config. Providing an option to enable user to specify a route as persistent as e.g:

`curl --request PATCH -H "Content-Type:application/json" -d '[{"cmd":"add", "ip_prefix":"1.1.1.1/32", "nexthop":"10.1.0.57", "persistent": true }]' http://127.0.0.1:8081/v1/config/vrf/default/routes`

If attribute is not set to true or not specified, the default is `false`.  Non-persistent route handling is as below:

1. Client **must** refresh the route at regular intervals within **3 minutes**
2. If route is not refreshed, the route is marked for deletion and gets removed in the next cycle. This could be anywhere between 3-6mts. 

**Server behavior:**
1. Route patch API must check if `persistent `flag is set to true. If so, it must write the entry to config DB - `STATIC_ROUTE` table
2. If flag is false/not-specified, write to APP DB - `STATIC_ROUTE `table
3. When the route is refreshed by the client, write the same _again_ to APP_DB instead of delete/add.

### CODE REVIEW

File: sonic_api.yaml [hunk 1]
```diff
@@ -2474,6 +2474,10 @@ definitions:
         type: integer
         format: int32
         description: vxlan id to be used for the tunnel. Optional arg. If it isn't provided the default vxlan id defined for the vnet/vrf will be used as the destination vnid.
+      persistent:
```
Comment (shi-su) [inline sonic_api.yaml:2477]: I was wondering what would be the expected behavior if there are requests for the same route prefix in both persistent and non-persistent mode? Also if there are entries for the same route prefix in both config_db and appl_db with the same distance, which one should take effect?

Comment (prsunny) [reply sonic_api.yaml:2477]: I did clarify that in the description. In normal cases, its not expected, but if it happens, persistent takes precedence

**Review by shi-su — APPROVED** [review]

**Comment by sumukhatv** [comment]
Should we add `persistent` to `delete` API too? https://github.com/Azure/sonic-restapi/blob/5cc4358fb67b9e2a0da9a6691064e41f97ebebc2/sonic_api.yaml#L1438

**Comment by prsunny** [comment]
> Should we add `persistent` to `delete` API too?
> 
> https://github.com/Azure/sonic-restapi/blob/5cc4358fb67b9e2a0da9a6691064e41f97ebebc2/sonic_api.yaml#L1438

I mentioned that in the 'persitent' flag description. Also thinking, for delete of route, we can delete it from both DBs pro-actively.

**Review by sumukhatv — APPROVED** [review]

**Pull request merged**